lms/templates/navigation-edx.html

```diff
@@ -142,7 +142,9 @@ <h1 class="logo" itemscope="" itemtype="http://schema.org/Organization">
 <![endif]-->
 % endif
 
-%if not user.is_authenticated():
+## The forgot password modal JavaScript assumes that JQuery is loaded,
+## which is not necessarily the case when using RequireJS.
+%if not enable_require_js and not user.is_authenticated():
   <%include file="forgot_password_modal.html" />
```

Comment:
I'm imagining an incremental approach, where specific pages will use RequireJS.  These pages would pass `enable_require_js = True` in the template context.  Two initial targets would be the combined login/registration page and the decoupled payment/verification flow, which were designed to eventually use RequireJS.